src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/IntellisenseQuickInfoBuilder.cs

```diff
@@ -86,9 +86,6 @@ internal static class IntellisenseQuickInfoBuilder
         }
 
         private static ClassifiedTextElement BuildClassifiedTextElement(QuickInfoSection section)
-        {
-            return new ClassifiedTextElement(section.TaggedParts.Select(
-                    part => new ClassifiedTextRun(part.Tag.ToClassificationTypeName(), part.Text)));
-        }
+            => Helpers.BuildClassifiedTextElement(section.TaggedParts);
```

Comment:
static which just forwards to another static. I'd prefer this method just be removed.  This stuff always just ends up confusing because some subset of callers one, both others call the other one.  And then something changes and they get out of sync.  Note: you can remove in a future PR.